cluster/addons/dns/README.md

```diff
@@ -29,6 +29,20 @@ Services, this resolves to the set of IPs of the pods selected by the Service.
 Clients are expected to consume the set or else use standard round-robin
 selection from the set.
 
+### SRV records
+SRV Records are created for Named ports that are part of "Normal" or Headless
```

Comment:
no need to capitalize "Named"

no need to quote "Normal" here - it was explained above.